app/assets/stylesheets/variables.scss

```diff
@@ -21,6 +21,7 @@ $yellow: #fefa87;
 $tan: #fdf9f3;
 $off-white: #fffffe;
 $bold-blue: #0045ff;
+$night-theme-markdown-inline-code: #ffffff;
```

Comment:
There are already variables which are defined by each theme. There is a var called `--theme-container-color` which defines `#fff` for the dark theme.